Sources/GRPC/Server.swift

```diff
@@ -352,9 +353,8 @@ extension Server {
     /// available to service providers via `context`. Defaults to a no-op logger.
     public var logger = Logger(label: "io.grpc", factory: { _ in SwiftLogNoOpLogHandler() })
 
-    /// Configuration for extracting trace IDs from metadata and injecting the extracted ID into
-    /// a logger.
-    public var traceIDExtractor: TraceIDExtractor?
+    /// Tracer that should when handling trace metadata.
+    public var tracer: (any Tracing.Tracer)? // TODO: tri-state "don't trace" / global / configured(any Tracer)
```

Comment:
Is 'configured' an idiomatic choice? I suppose it's probably helpful for testing.

Reply:
I think it might be, similar to how some frameworks pass around metrics factories to carry around a testing one but not lose the ability to run tests in parallel etc.

It'd be a triple though: "default == use whatever global configured thing is", "configured(Tracer) - for e.g. testing", or "ignore" I kind of wonder if it even makes sense but we can allow it I suppose.